internal/resolver/dns/dns_resolver_test.go

```diff
@@ -764,103 +764,113 @@ func TestDNSResolverExponentialBackoff(t *testing.T) {
 		return t
 	}
 	tests := []struct {
+		name     string
 		target   string
 		addrWant []resolver.Address
 		scWant   string
 	}{
 		{
+			"happy-case-default-port",
```

Comment:
FWIW/future references you can use spaces instead of dashes; the test framework will replace them (with underscores i think?).  That would make them a little easier to read IMO, but you don't need to change them.

Reply:
Sounds good. Thanks for the heads up.

Reply:
Changed.